server/backroundJob/clearCardsCol.js

```diff
@@ -0,0 +1,41 @@
+const cron = require('node-cron');
+const query = require('../database/mongodb.js');
+const utils = require('../utils.js');
+
+// this job triggers every 12h to clean the expired cards and move them to expired collection
+const expiredCardsCleaner = () => {
+  cron.schedule('12  * * * *', () => {
+    query.findAllIn('cards', { expDate: { $exists: true } }, (err, res) => {
+      if (err) return err;
+
+      const validCards = [];
+      const expiredCards = [];
+
+      res.map((card) => {
+        if (utils.isValid(card.expDate)) {
+          validCards.push(card);
+        } else expiredCards.push(card);
```

Comment:
Would be great if this is done without a `push` as it isn't pure. What do you think?